.vscode/launch.json

```diff
@@ -1,58 +1,53 @@
 {
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Jest Tests",
-            "type": "node",
-            "request": "launch",
-            "args": [],
-            "cwd": "${workspaceFolder}/static",
-            "console": "integratedTerminal",
-            "program": "${workspaceFolder}/static/node_modules/jest/bin/jest",
-        },
-        {
-            "name": "Flask (with NL)",
-            "type": "python",
-            "request": "launch",
-            "stopOnEntry": false,
-            "python": "${workspaceFolder}/.env/bin/python3",
-            "program": "${workspaceRoot}/web_app.py",
-            "env": {
-                "FLASK_APP": "${workspaceRoot}/web_app.py",
-                "FLASK_ENV": "local",
-                "FLASK_DEBUG": "1",
-                "ENABLE_MODEL": "true"
-            },
-            "args": [
-                "8080",
-                "--no-debugger",
-            ],
-            "cwd": "${workspaceRoot}",
-            "envFile": "${workspaceFolder}/.env",
-            "jinja": true,
-            "console": "internalConsole",
-        },
-        {
-            "name": "Flask (without NL)",
-            "type": "python",
-            "request": "launch",
-            "stopOnEntry": false,
-            "python": "${workspaceFolder}/.env/bin/python3",
-            "program": "${workspaceRoot}/web_app.py",
-            "env": {
-                "FLASK_APP": "${workspaceRoot}/web_app.py",
-                "FLASK_ENV": "local",
-                "FLASK_DEBUG": "1",
-                "ENABLE_MODEL": "false"
-            },
-            "args": [
-                "8080",
-                "--no-debugger",
-            ],
-            "cwd": "${workspaceRoot}",
-            "envFile": "${workspaceFolder}/.env",
-            "jinja": true,
-            "console": "internalConsole",
-        }
-    ]
-}
\ No newline at end of file
+  "version": "0.2.0",
```

Comment:
Do you intend to change this?

Reply:
Yes, I found my local debug complains about gcloud project, adding this fix the issue.